src/components/ObjectForm.vue

```diff
@@ -184,11 +184,15 @@ export default {
       needsRecordType: computed(() => {
         return !state.recordTypeId
       }),
+      fieldValidationErrors: [],
```

Comment:
I 'd suggest to make this one computed and just read its length to determine `isValid`. Side-effects in computed are an antipattern. I know it's a dying project but it looks like we 'll maintain it for a while.

Reply:
> I 'd suggest to make this one computed and just read its length to determine `isValid`. Side-effects in computed are an antipattern. I know it's a dying project but it looks like we 'll maintain it for a while.

But the list of fields is needed, which are required and invalid. Not only valid true/false

Reply:
```js
      fieldValidationErrors: computed(() => {
        const errors = []

        state.fields.forEach((field) => {
          if (!field.valid(state.obj[field.name]) && field.updateable) {
            errors.push(`${field.label} is mandatory.`)
          }
        })

        return errors
      }),
      isValid: state.fieldValidationErrors.length === 0,
```
something like that was my idea, but as you said, it might affect other projects. just my 2 cents.